<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDataPlot</class>
 <widget class="QWidget" name="frmDataPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="plotwidget" native="true"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cbIsWaveVisible">
       <property name="text">
        <string>显示波形</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbIsGridVisible">
       <property name="text">
        <string>显示栅格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbIsMinorGridVisible">
       <property name="text">
        <string>显示小栅格</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbIsDarkBackground">
       <property name="text">
        <string>黑色背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbTestMode">
       <property name="text">
        <string>测试模式</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnChangeColor">
       <property name="text">
        <string>波 形 颜 色</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnUnzoom">
       <property name="text">
        <string>恢 复 缩 放</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnSaveWave">
       <property name="text">
        <string>保 存 波 形</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
